src/app.js

```diff
@@ -276,34 +281,81 @@ fluid.onUncaughtException.addListener(function (err) {
  */
 
 fluid.defaults("gpii.app.tray", {
-    gradeNames: "fluid.component",
+    gradeNames: "fluid.modelComponent",
     members: {
         tray: {
             expander: {
                 funcName: "gpii.app.makeTray",
-                args: ["{that}.options.icon"]
+                args: ["{that}.model.icon"]
             }
+        },
+        icons: {
```

Comment:
Is there a reason to put this in 'members' rather than directly in the 'options' structure?

Reply:
Hmm, it really sounds reasonable for the icons to be configurable. We'll update it according to your advice :)

Reply:
It's still configurable in the options block, isn't it? Am I missing something about how you'd like to configure it?